src/components/SkillPage/SkillListing.js

```diff
@@ -138,6 +158,18 @@ class SkillListing extends Component {
 
     };
 
+    saveSkillRatings = (skill_ratings) => {
+        // Sample data
+        const data = [{name: '5 ⭐', value:20},
+              {name: '4 ⭐', value:14},
+              {name: '3 *', value:16},
```

Comment:
I think we should have a star symbol there, since it is shown in - https://developers.google.com/actions/distribute/directory

Thought I think we can use a better icon there rather than this unicode icon.

 What do you think @akshatnitd ? :thinking:

Reply:
Yes, instead of `*`, an icon can be used